src/git/from_vcs.py

```diff
@@ -14,7 +14,7 @@ def run_command(): pass  # --STRIP DURING BUILD
 class NotThisMethod(Exception):  # --STRIP DURING BUILD
     pass  # --STRIP DURING BUILD
 @register_vcs_handler("git", "pieces_from_vcs")
-def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
+def git_pieces_from_vcs(tag_prefix, root, verbose, r_c=run_command):
```

Comment:
FWIW I don't think `r_c` is clearer than `run_command`. What issue is being resolved by this one?

Reply:
/_version.py:219:51: W0621: Redefining name 'run_command' from outer scope (line 70) (redefined-outer-name)
Yeah I agree r_c is not clear, I'm still thinking about this.

Reply:
Offhand suggestion (I am not well-versed in the details): rename the outer function to `default_run_command`

Reply:
What precisely do you mean by 'suppressed in a single place'?

Reply:
I think @bryevdv meant adding something like `#noqa` to this line, or some other marker that would hopefully avoid the problem.

What about:

```suggestion
def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
```